### PR TITLE
Minor changes Portuguese Brazilian

### PR DESCRIPTION
**Pull request by bitcoinuser**

### CODE REVIEW

File: src/common/translations/pt-BR.json [hunk 1]
```diff
@@ -126,28 +126,28 @@
...
         "emptyWalletError": "Carteira vazia",
-        "selectMainnetAsset": "Selecione uma carteira que tenha pelo menos um ativo na rede principal da rsk",
-        "selectTestnetAsset": "Selecione uma carteira que tenha pelo menos um ativo na rede de testes da rsk",
+        "selectMainnetAsset": "Selecione uma carteira que tenha pelo menos um ativo na rede principal da Rsk",
+        "selectTestnetAsset": "Selecione uma carteira que tenha pelo menos um ativo na rede de testes da Rsk",
         "isConnecting": "Conectando carteira...",
         "isSigning": "Assinando transação..",
         "signApproved": "Assinatura de mensagem aprovada",
-        "signApprovedDesc": "Sucesso!Sua mensagem foi assinada e deve aparecer no dapp",
+        "signApprovedDesc": "Sucesso!Sua mensagem foi assinada e deve aparecer no Dapp",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:134]:
```suggestion
        "signApprovedDesc": "Sucesso!Sua mensagem foi assinada e deve aparecer no dapp",
```

**Review by chrisli30 — CHANGES_REQUESTED** [review]
Sorry, what's the reason to update RSK to Rsk?

File: src/common/translations/pt-BR.json [hunk 2]
```diff
@@ -126,28 +126,28 @@
         "timeoutBody": "A RWallet não conseguiu estabelecer uma conexão Wallet Connect com o Dapp dentro do prazo estabelecido. Por favor, tente novamente.",
         "timeoutButton": "Ok",
         "emptyWalletError": "Carteira vazia",
-        "selectMainnetAsset": "Selecione uma carteira que tenha pelo menos um ativo na rede principal da rsk",
-        "selectTestnetAsset": "Selecione uma carteira que tenha pelo menos um ativo na rede de testes da rsk",
+        "selectMainnetAsset": "Selecione uma carteira que tenha pelo menos um ativo na rede principal da Rsk",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:129]:
```suggestion
        "selectMainnetAsset": "Selecione uma carteira que tenha pelo menos um ativo na rede principal da RSK",
```

File: src/common/translations/pt-BR.json [hunk 3]
```diff
@@ -126,28 +126,28 @@
         "timeoutBody": "A RWallet não conseguiu estabelecer uma conexão Wallet Connect com o Dapp dentro do prazo estabelecido. Por favor, tente novamente.",
         "timeoutButton": "Ok",
         "emptyWalletError": "Carteira vazia",
-        "selectMainnetAsset": "Selecione uma carteira que tenha pelo menos um ativo na rede principal da rsk",
-        "selectTestnetAsset": "Selecione uma carteira que tenha pelo menos um ativo na rede de testes da rsk",
+        "selectMainnetAsset": "Selecione uma carteira que tenha pelo menos um ativo na rede principal da RSK",
+        "selectTestnetAsset": "Selecione uma carteira que tenha pelo menos um ativo na rede de testes da Rsk",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:130]:
```suggestion
        "selectTestnetAsset": "Selecione uma carteira que tenha pelo menos um ativo na rede de testes da RSK",
```

File: src/common/translations/pt-BR.json [hunk 4]
```diff
@@ -181,7 +181,7 @@
         "data": "Dados",
         "networkQuestion": {
           "title": "Rede",
-          "body": "A rede de testes ou testnet é uma blockchain alternativa da RSK usada exclusivamente para realização de testes"
+          "body": "A rede de testes ou testnet é uma blockchain alternativa da Rsk usada exclusivamente para realização de testes"
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:184]:
```suggestion
          "body": "A rede de testes ou testnet é uma blockchain alternativa da RSK usada exclusivamente para realização de testes"
```

File: src/common/translations/pt-BR.json [hunk 5]
```diff
@@ -289,7 +289,7 @@
     },
     "mine": {
       "index": {
-        "joinRSKCommunity": "Junte-se à comunidade RSK",
+        "joinRSKCommunity": "Junte-se à comunidade Rsk",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:292]:
```suggestion
        "joinRSKCommunity": "Junte-se à comunidade RSK",
```

File: src/common/translations/pt-BR.json [hunk 6]
```diff
@@ -623,10 +623,10 @@
       "body": "Ocorreu um erro ao calcular o custo. Por favor, tente novamente."
     },
     "invalidRskAddress": {
-      "title": "Endereço RSK inválido",
+      "title": "Endereço Rsk inválido",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:626]:
```suggestion
      "title": "Endereço RSK inválido",
```

File: src/common/translations/pt-BR.json [hunk 7]
```diff
@@ -625,8 +625,8 @@
     "invalidRskAddress": {
       "title": "Endereço RSK inválido",
       "body": {
-        "normal": "O endereço inserido é um endereço RSK inválido. Certifique-se de que o endereço do destinatário é um endereço RSK válido e verifique-o em %{explorerName} antes de enviar.",
-        "ethereum": "O endereço inserido parece ser um endereço Ethereum. Este é um problema conhecido ao copiar endereços do MetaMask. Certifique-se de que o endereço do destinatário é um endereço RSK válido e verifique-o em %{explorerName} antes de enviar."
+        "normal": "O endereço inserido é um endereço Rsk inválido. Certifique-se de que o endereço do destinatário é um endereço Rsk válido e verifique-o em %{explorerName} antes de enviar.",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:628]:
```suggestion
        "normal": "O endereço inserido é um endereço RSK inválido. Certifique-se de que o endereço do destinatário é um endereço RSK válido e verifique-o em %{explorerName} antes de enviar.",
```

File: src/common/translations/pt-BR.json [hunk 8]
```diff
@@ -639,17 +639,17 @@
     "emptyRecordAddress": {
       "title": "Endereço sem registro",
       "body": {
-        "normal": "O endereço inserido não tem registro em %{explorerName} e tem %{balance} %{otherSymbolName} na rede RSK %{otherType}. Tem certeza de que deseja enviar %{amount} %{symbolName} para o endereço RSK da %{type}?",
-        "both": "O endereço inserido não tem registro em %{explorerName}. Tem certeza de que deseja enviar %{amount} %{symbolName} para o endereço RSK na %{type}?"
+        "normal": "O endereço inserido não tem registro em %{explorerName} e tem %{balance} %{otherSymbolName} na rede Rsk %{otherType}. Tem certeza de que deseja enviar %{amount} %{symbolName} para o endereço Rsk da %{type}?",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:642]:
```suggestion
        "normal": "O endereço inserido não tem registro em %{explorerName} e tem %{balance} %{otherSymbolName} na rede RSK %{otherType}. Tem certeza de que deseja enviar %{amount} %{symbolName} para o endereço RSK da %{type}?",
```

File: src/common/translations/pt-BR.json [hunk 9]
```diff
@@ -640,16 +640,16 @@
...
       "body": {
         "normal": "O endereço inserido não tem registro em %{explorerName} e tem %{balance} %{otherSymbolName} na rede RSK %{otherType}. Tem certeza de que deseja enviar %{amount} %{symbolName} para o endereço RSK da %{type}?",
-        "both": "O endereço inserido não tem registro em %{explorerName}. Tem certeza de que deseja enviar %{amount} %{symbolName} para o endereço RSK na %{type}?"
+        "both": "O endereço inserido não tem registro em %{explorerName}. Tem certeza de que deseja enviar %{amount} %{symbolName} para o endereço Rsk na %{type}?"
       }
     },
     "convertWalletAddress": {
-      "title": "Endereço da carteira RSK inválido",
-      "body": "O endereço da carteira RSK inserido é inválido. A RWallet tentará corrigi-lo automaticamente para o formato de endereço correto. Você deseja prosseguir?"
+      "title": "Endereço da carteira Rsk inválido",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:647]:
```suggestion
      "title": "Endereço da carteira RSK inválido",
```

File: src/common/translations/pt-BR.json [hunk 10]
```diff
@@ -640,16 +640,16 @@
...
       "body": {
         "normal": "O endereço inserido não tem registro em %{explorerName} e tem %{balance} %{otherSymbolName} na rede RSK %{otherType}. Tem certeza de que deseja enviar %{amount} %{symbolName} para o endereço RSK da %{type}?",
-        "both": "O endereço inserido não tem registro em %{explorerName}. Tem certeza de que deseja enviar %{amount} %{symbolName} para o endereço RSK na %{type}?"
+        "both": "O endereço inserido não tem registro em %{explorerName}. Tem certeza de que deseja enviar %{amount} %{symbolName} para o endereço Rsk na %{type}?"
       }
     },
     "convertWalletAddress": {
       "title": "Endereço da carteira RSK inválido",
-      "body": "O endereço da carteira RSK inserido é inválido. A RWallet tentará corrigi-lo automaticamente para o formato de endereço correto. Você deseja prosseguir?"
+      "body": "O endereço da carteira Rsk inserido é inválido. A RWallet tentará corrigi-lo automaticamente para o formato de endereço correto. Você deseja prosseguir?"
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:648]:
```suggestion
      "body": "O endereço da carteira RSK inserido é inválido. A RWallet tentará corrigi-lo automaticamente para o formato de endereço correto. Você deseja prosseguir?"
```

File: src/common/translations/pt-BR.json [hunk 11]
```diff
@@ -640,16 +640,16 @@
...
       }
     },
     "convertWalletAddress": {
       "title": "Endereço da carteira RSK inválido",
       "body": "O endereço da carteira RSK inserido é inválido. A RWallet tentará corrigi-lo automaticamente para o formato de endereço correto. Você deseja prosseguir?"
     },
     "convertContractAddress": {
-      "title": "Endereço de contrato RSK inválido",
-      "body": "O endereço do contrato RSK inserido é inválido. A RWallet tentará corrigi-lo automaticamente para o formato de endereço correto. Você deseja prosseguir?"
+      "title": "Endereço de contrato Rsk inválido",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:651]:
```suggestion
      "title": "Endereço de contrato RSK inválido",
```

File: src/common/translations/pt-BR.json [hunk 12]
```diff
@@ -649,7 +649,7 @@
     },
     "convertContractAddress": {
       "title": "Endereço de contrato RSK inválido",
-      "body": "O endereço do contrato RSK inserido é inválido. A RWallet tentará corrigi-lo automaticamente para o formato de endereço correto. Você deseja prosseguir?"
+      "body": "O endereço do contrato Rsk inserido é inválido. A RWallet tentará corrigi-lo automaticamente para o formato de endereço correto. Você deseja prosseguir?"
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:652]:
```suggestion
      "body": "O endereço do contrato RSK inserido é inválido. A RWallet tentará corrigi-lo automaticamente para o formato de endereço correto. Você deseja prosseguir?"
```

File: src/common/translations/pt-BR.json [hunk 13]
```diff
@@ -640,7 +640,7 @@
       "title": "Endereço sem registro",
       "body": {
         "normal": "O endereço inserido não tem registro em %{explorerName} e tem %{balance} %{otherSymbolName} na rede RSK %{otherType}. Tem certeza de que deseja enviar %{amount} %{symbolName} para o endereço RSK da %{type}?",
-        "both": "O endereço inserido não tem registro em %{explorerName}. Tem certeza de que deseja enviar %{amount} %{symbolName} para o endereço RSK na %{type}?"
+        "both": "O endereço inserido não tem registro em %{explorerName}. Tem certeza de que deseja enviar %{amount} %{symbolName} para o endereço Rsk na %{type}?"
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:643]:
```suggestion
        "both": "O endereço inserido não tem registro em %{explorerName}. Tem certeza de que deseja enviar %{amount} %{symbolName} para o endereço RSK na %{type}?"
```

File: src/common/translations/pt-BR.json [hunk 14]
```diff
@@ -626,7 +626,7 @@
       "title": "Endereço RSK inválido",
       "body": {
         "normal": "O endereço inserido é um endereço RSK inválido. Certifique-se de que o endereço do destinatário é um endereço RSK válido e verifique-o em %{explorerName} antes de enviar.",
-        "ethereum": "O endereço inserido parece ser um endereço Ethereum. Este é um problema conhecido ao copiar endereços do MetaMask. Certifique-se de que o endereço do destinatário é um endereço RSK válido e verifique-o em %{explorerName} antes de enviar."
+        "ethereum": "O endereço inserido parece ser um endereço Ethereum. Este é um problema conhecido ao copiar endereços do MetaMask. Certifique-se de que o endereço do destinatário é um endereço Rsk válido e verifique-o em %{explorerName} antes de enviar."
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:629]:
```suggestion
        "ethereum": "O endereço inserido parece ser um endereço Ethereum. Este é um problema conhecido ao copiar endereços do MetaMask. Certifique-se de que o endereço do destinatário é um endereço RSK válido e verifique-o em %{explorerName} antes de enviar."
```

**Comment by bitcoinuser** [comment]
> Sorry, what's the reason to update RSK to Rsk?

Ok, changed to RSK

File: src/common/translations/pt-BR.json [hunk 15]
```diff
@@ -121,33 +121,33 @@
...
         "timeoutButton": "Ok",
         "emptyWalletError": "Carteira vazia",
-        "selectMainnetAsset": "Selecione uma carteira que tenha pelo menos um ativo na rede principal da rsk",
-        "selectTestnetAsset": "Selecione uma carteira que tenha pelo menos um ativo na rede de testes da rsk",
+        "selectMainnetAsset": "Selecione uma carteira que tenha pelo menos um ativo na rede principal da RSK",
+        "selectTestnetAsset": "Selecione uma carteira que tenha pelo menos um ativo na rede de testes da RSK",
         "isConnecting": "Conectando carteira...",
         "isSigning": "Assinando transação..",
         "signApproved": "Assinatura de mensagem aprovada",
         "signApprovedDesc": "Sucesso!Sua mensagem foi assinada e deve aparecer no dapp",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:134]:
```suggestion
        "signApprovedDesc": "Sucesso! Sua mensagem foi assinada e deve aparecer no dapp",
```

File: src/common/translations/pt-BR.json [hunk 16]
```diff
@@ -121,33 +121,33 @@
         "unauthorization": "Por favor, ative as permissões da câmera para usar esse recurso."
       },
       "walletconnect": {
-        "title": "Wallet Connect",
+        "title": "WalletConnect",
         "timeoutTitle": "Tempo expirado",
-        "timeoutBody": "A RWallet não conseguiu estabelecer uma conexão Wallet Connect com o Dapp dentro do prazo estabelecido. Por favor, tente novamente.",
+        "timeoutBody": "A RWallet não conseguiu estabelecer uma conexão WalletConnect com o Dapp dentro do prazo estabelecido. Por favor, tente novamente.",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:126]:
```suggestion
        "timeoutBody": "A RWallet não conseguiu estabelecer uma conexão WalletConnect com o dapp dentro do prazo estabelecido. Por favor, tente novamente.",
```

File: src/common/translations/pt-BR.json [hunk 17]
```diff
@@ -159,10 +159,10 @@
         "cancel": "Cancelar",
         "gotIt": "ENTENDI",
         "leaveToDisconnect": "Sair para desconectar?",
-        "leaveToDisconnectDesc": "Sair da página Wallet Connect irá fazer com que a RWallet se desconecte do Dapp atualmente conectado.",
+        "leaveToDisconnectDesc": "Sair da página WalletConnect irá fazer com que a RWallet se desconecte do Dapp atualmente conectado.",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:162]:
```suggestion
        "leaveToDisconnectDesc": "Sair da página WalletConnect irá fazer com que a RWallet se desconecte do dapp atualmente conectado.",
```

File: src/common/translations/pt-BR.json [hunk 18]
```diff
@@ -159,10 +159,10 @@
...
+        "leaveToDisconnectDesc": "Sair da página WalletConnect irá fazer com que a RWallet se desconecte do dapp atualmente conectado.",
         "leaveToDisconnectConfirm": "Você tem certeza de que deseja desconectar?",
-        "disconnectAlertTitle": "Wallet Connect desconectada",
-        "disconnectAlertContent": "O Dapp \"%{dappName}\" foi desconectado da Wallet Connect. Pode ter ocorrido instabilidade no acesso a rede ou rejeição por parte do Dapp. Ao clicar no botão abaixo você será direcionado para a carteira.",
+        "disconnectAlertTitle": "WalletConnect desconectada",
+        "disconnectAlertContent": "O dapp \"%{dappName}\" foi desconectado da WalletConnect. Pode ter ocorrido alguma instabilidade ou rejeição por parte do dapp. Ao clicar no botão abaixo você será direcionado para a carteira.",
         "okGotIt": "Ok, entendi",
         "error": "Erro",
         "errorDesc": "Erro!Aconteceu um erro, por favor tente novamente mais tarde (pode haver muitas razões, como problemas de rede, rejeição do usuário, código de senha incorreto, taxa insuficiente, etc.)",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:168]:
```suggestion
        "errorDesc": "Erro! Aconteceu um erro, por favor tente novamente mais tarde (pode haver muitas razões, como problemas de rede, rejeição do usuário, código de senha incorreto, taxa insuficiente, etc.)",
```

**Comment by chrisli30** [comment]
@bitcoinuser Sorry, you need to merge all those commits into one and re-submit a PR. There are just too many commits in this PR.

**Pull request closed without merging**